airqo-platform/AirQo-frontend
# [NetManager] Airqo dashboard

#### Summary of Changes (What does this PR do?)
- Enable AirQo dashboard
- The averages and exceedances graphs are currently deactivated

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch and checkout to this branch locally
* Install requirements `npm install`
* Ensure this backend [PR](https://github.com/airqo-platform/AirQo-api/pull/177) (new analytics microservice) is running
* Set the correct `analytics` Base_URL in the frontend `.env` file.
* Start netmanager application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* You should be able to access dash board on the AirQo org too

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)
